### PR TITLE
cleanup after RDP release

### PR DESCRIPTION
**Pull request by trilopin**
#### Summary
Cleanup after new RDP release:
- removed ignored old e2e spec 
- moved timeline and retro to from playbook_run_backstage to playbook_run
- header button done internally, fixed a clause for size in the styled component
- removed orphan duplicated components from old playbook_run (retrospective, profile_vertical, etc...)

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-45451

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

**Comment by trilopin** [comment]
There's a regression in headerbutton, will fix it before asking review

**Review by crspeller — APPROVED** [review]

File: webapp/src/components/backstage/playbook_runs/playbook_run/header_button.tsx [hunk 1]
```diff
@@ -5,9 +5,14 @@ import styled, {css} from 'styled-components';
...
 
 import Tooltip from 'src/components/widgets/tooltip';
-import {HeaderIcon} from '../playbook_run_backstage/playbook_run_backstage';
 import {CompassIcon} from 'src/types/compass';
 
+declare module 'react-bootstrap/esm/OverlayTrigger' {
+    interface OverlayTriggerProps {
+        shouldUpdatePosition?: boolean;
+    }
+}
```
Comment (gbochora) [inline webapp/src/components/backstage/playbook_runs/playbook_run/header_button.tsx:14]: I couldn't figure out what is for `OverlayTriggerProps`? Do we use it?

**Review by gbochora — APPROVED** [review]
Great! 🎉

File: webapp/src/components/backstage/playbook_runs/playbook_run/header_button.tsx (hunk 1 above)
Comment (trilopin) [reply webapp/src/components/backstage/playbook_runs/playbook_run/header_button.tsx:14]: I moved from the old backstage run page to this new one the headerButton component. That component used shouldUpdatePosition prop. Apparently, this change is needed for telling typescript that we can pass that prop.

It was in the previous one, and I had to move it here.

**Review by gbochora — APPROVED** [review]
Looks good!

**Pull request merged**